man/man5/zpool-features.5

```diff
@@ -25,7 +26,8 @@ zpool\-features \- ZFS pool feature descriptions
 ZFS pool on\-disk format versions are specified via "features" which replace
 the old on\-disk format numbers (the last supported on\-disk format number is
 28). To enable a feature on a pool use the \fBupgrade\fR subcommand of the
-zpool(8) command, or set the \fBfeature@\fR\fIfeature_name\fR property
+zpool(8) command (but note the operation of the \fBcompatibility\fR property in
+zpoolprops(8)), or set the \fBfeature@\fR\fIfeature_name\fR property
 to \fBenabled\fR.
 .sp
 .LP
```

Comment:
I think there should be a short introduction to compatibility files here, like:

`For ease of administration, commonly-used sets of features can be specified using compatibility files. By default, compatibility files are provided for popular distributions that use OpenZFS and for features supported in all Tier-1 platforms on January 1st of each year.`

Reply:
ACK, thanks.